gitpod-io/gitpod
# fix: update gateway instructions

## Description

Update gateway instructions from repo to main documentation. 

## Related Issue(s)

N/A

## How to test

- Set JetBrains preferences
- Open workspace
- Links should be as expected (link to docs, not to GitHub / markdown)

## Release Notes

```release-note
```

## Documentation

N/A
